oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexHelper.java

```diff
@@ -194,6 +197,18 @@ private static void mapInternalProperties(XContentBuilder mappingBuilder) throws
         // .endObject();
     }
 
+    private static void createNestedAnalyzed(XContentBuilder mappingBuilder) throws IOException {
```

Comment:
I would change the name to something like `createAnalyzedSubfield`. We should not use `Nested` since it can be confused with the nested type in elasticsearch that we use widely but not in this case.

Reply:
Done.